Sources/SubviewsSkeletonables.swift

```diff
@@ -14,7 +14,8 @@ extension UIView {
 
 extension UITableView {
     override var subviewsToSkeleton: [UIView] {
-        visibleCells + visibleSectionHeaders + visibleSectionFooters
+		guard window != nil else { return [] }
```

Comment:
I think this could be a good workaround. Because of that, I think we should include a comment here, explaining why we are checking if the `window` is `nil`. WDYT?

Reply:
@Juanpe I now updated PR. Please check. thanks

Reply:
I think is enough :) Thx